### PR TITLE
Submitting PC_DIMMER version 5.6.0

### PR DESCRIPTION
**Pull request by xn--nding-jua**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/68367)

### CODE REVIEW

**Comment by xn--nding-jua** [comment]
@msftbot the set URL to the installer is correct and is working.

File: manifests/p/PHOENIXstudios/PC_DIMMER/5.6.0/PHOENIXstudios.PC_DIMMER.locale.de-DE.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS0.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: PHOENIXstudios.PC_DIMMER
+PackageVersion: 5.6.0
+PackageLocale: de-DE
+Publisher: PHOENIXstudios
```
Comment (Trenly) [inline manifests/p/PHOENIXstudios/PC_DIMMER/5.6.0/PHOENIXstudios.PC_DIMMER.locale.de-DE.yaml:7]:
```suggestion
Publisher: Dr.-Ing. Christian Nöding
```
This must match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/p/PHOENIXstudios/PC_DIMMER/5.6.0/PHOENIXstudios.PC_DIMMER.locale.de-DE.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+PublisherUrl: https://www.pcdimmer.de
+PublisherSupportUrl: https://www.pcdimmer.de/phpBB3
+PrivacyUrl: https://www.pcdimmer.de/index.php/home/datenschutz
+Author: Dr.-Ing. Christian Nöding
+PackageName: PC_DIMMER
+PackageUrl: https://www.pcdimmer.de
+License: GPL
```
[the remark this answers is not in the document]
Comment (xn--nding-jua) [reply manifests/p/PHOENIXstudios/PC_DIMMER/5.6.0/PHOENIXstudios.PC_DIMMER.locale.de-DE.yaml:14]: Hi, PC_DIMMER uses GPLv2. I will change the commit.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**